tutorials/katacoda/thanos/1-globalview/step1.md

```diff
@@ -70,6 +70,8 @@ scrape_configs:
       - targets: ['127.0.0.1:9091','127.0.0.1:9092']
 </pre>
 
+**NOTE** : The `global` entry should have `external_labels` else Thanos will not recognise it.
```

Comment:
I think we can be more helpful and this and describe it more. What do you think?

Think about someone new here. What it means `recognise` here? What if it's empty? What can go wrong? :hugs:

Reply:
:+1: I think let's describe why do we need this, what this does, the effects of deduplication.